cmd/cluster-ingress-operator/main.go

```diff
@@ -4,88 +4,104 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"runtime"
 	"strings"
-	"time"
 
-	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/cluster-ingress-operator/pkg/dns"
 	awsdns "github.com/openshift/cluster-ingress-operator/pkg/dns/aws"
-	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
 	"github.com/openshift/cluster-ingress-operator/pkg/operator"
-	stub "github.com/openshift/cluster-ingress-operator/pkg/stub"
+	operatorconfig "github.com/openshift/cluster-ingress-operator/pkg/operator/config"
 	"github.com/openshift/cluster-ingress-operator/pkg/util"
 
-	"github.com/operator-framework/operator-sdk/pkg/k8sclient"
-	sdk "github.com/operator-framework/operator-sdk/pkg/sdk"
-	k8sutil "github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
```

Comment:
Looking at the changes, operator-sdk pkg is used only here, mainly to get `WATCH_NAMESPACE` env. 
Why not use `os.LookupEnv(...)` directly and get rid of operator-sdk pkg dependency?

Reply:
Good point. I'd actually also consider switching to a Cobra command and consuming this from flags anyway. In either case, seems like a followup?

Reply:
yes, that works